.github/workflows/ci.yml

```diff
@@ -11,15 +11,14 @@ on:
 jobs:
   build:
     name: ghc ${{ matrix.ghc }}
-    runs-on: ubuntu-16.04
+    runs-on: ubuntu-latest
     strategy:
       matrix:
-        ghc: ["8.2.2", "8.4.4", "8.6.5", "8.8.3", "8.10.1", "9.0.1"]
+        ghc: ["8.2.2", "8.4.4", "8.6.5", "8.8.3", "8.10.1", "9.0.1", "9.2"]
         cabal: ["latest"]
 
     steps:
     - uses: actions/checkout@v2
-      if: github.event.action == 'opened' || github.event.action == 'synchronize' || github.event.ref == 'refs/heads/master'
 
     - uses: haskell/actions/setup@v1.1.8
```

Comment:
I think what I did for `fused-effects` was [tell it to use v1](https://github.com/fused-effects/fused-effects/blob/a0eae849f0538591a733b1a8ebe98ac866f3885e/.github/workflows/ci.yml#L22) instead of some more precise version, and then do the same for all [the ghc versions](https://github.com/fused-effects/fused-effects/blob/a0eae849f0538591a733b1a8ebe98ac866f3885e/.github/workflows/ci.yml#L17).